packages/react/src/dialog-footer/DialogFooter.tsx

```diff
@@ -0,0 +1,26 @@
+// import * as RadixDialog from "@radix-ui/react-dialog";
+import { type ComponentPropsWithRef, type ReactNode, forwardRef } from "react";
+
+import type { ExtendProps } from "../utils";
+
+import { Flex } from "../flex";
+import * as styles from "./DialogFooter.css";
+
+type FooterProps = ExtendProps<
+  ComponentPropsWithRef<typeof Flex>,
+  {
+    children: ReactNode;
+  }
+>;
```

Comment:
we can simplify this
```suggestion
type FooterProps = ComponentPropsWithRef<typeof Flex>;
```